community/import-tool/src/test/java/org/neo4j/tooling/QuickImport.java

```diff
@@ -161,6 +161,7 @@ public long maxMemoryUsage()
             }
             else
             {
+                System.out.println( "Seed " + randomSeed );
```

Comment:
looks like debug output

Reply:
QuickImport is a test utility, lives in src/test. It now prints the seed it uses to generate its data, which is very helpful for debugging.

Reply:
oh, please nevermind then